eventhandling/configureEvent.go

```diff
@@ -191,11 +192,11 @@ func (eh ConfigureMonitoringEventHandler) updatePrometheusConfigMap(eventData ke
 		// (a) if a scrape config with the same name is available, update that one
 
 		// <service>-primary.<project>-<stage>
-		createScrapeJobConfig(scrapeConfig, config, eventData.Project, stage.Name, eventData.Service, false, true)
+		createScrapeJobConfig(scrapeConfig, config, eventData.Project, stage.Name, eventData.Service, false, true, eh.logger)
```

Comment:
Can we parse the duration (including the default and the logging) at the beginning of `updatePrometheusConfigMap` instead of pushing down a keptn Logger ?
It would simplify the function logic and the signature would be cleaner with a `Duration` value being passed instead of a logger and an access to some global value happening within the function body

Reply:
Yes, that would probably improve the code quite a bit.